docs/en/enterprise-edition/content-collections/application-security/ides/connect-vscode.adoc

```diff
@@ -40,9 +40,9 @@ NOTE: The plugin automatically invokes the latest version of Prisma Cloud Code S
 
 .. For *GRBAC*, provide the following permissions.
 +
-* *Policies*: 'View' permissions
-* *Projects*: 'View' permissions
-* *Repositories*: ('View' and 'Create') OR ('View' and 'Update') permissions
+* *Policies* -> *Policies*: 'View' permissions
+* *Application Security* -> *Projects*: 'View' permissions
+* *Settings* -> *Providers*: ('View' and 'Create') OR ('View' and 'Update') permissions
```

Comment:
What are "Providers" in this context? Code providers? If so, state the entity requiring the permissions, the permissions required and then the path.

Reply:
<img width="860" alt="Screenshot 2024-10-27 at 10 45 48 PM" src="https://github.com/user-attachments/assets/b385fc6e-f1a3-40c4-ad29-f199b79c6f85">

Reply:
I would have the name of the entity requiring the permission, the permission, and finally the path. For example: Providers: CRUD. To configure these permissions, navigate to Settings > Providers, and select the required permissions.